lib/hterm.js

```diff
@@ -1,16 +1,42 @@
 import {hterm, lib} from 'hterm-umdjs';
+import unicodeStringUtils from 'unicode-string-utils';
```

Comment:
@dotcypress Hey, thanks for the feedback. Can you provide me with steps that produce unexpected result because of unicodeStringUtils? I haven't been able to create a problematic test case on my computer.